wagtail/core/templatetags/wagtailcore_tags.py

```diff
@@ -11,11 +12,17 @@
 
 
 @register.simple_tag(takes_context=True)
-def pageurl(context, page):
+def pageurl(context, page, **kwargs):
```

Comment:
`def pageurl(context, page, fallback=None)` would be better here, instead of retrieving `fallback` from `kwargs` - this way, we avoid silently accepting all keyword arguments, and users will get the expected error "'pageurl' received unexpected keyword argument" if they misspell `fallback` (for example).